src/osd/PeeringState.cc

```diff
@@ -6227,6 +6227,13 @@ boost::statechart::result PeeringState::Active::react(const AllReplicasActivated
     pl->send_pg_created(pgid);
   }
 
+  psdout(1) << __func__ << " AllReplicasActivated : Peering->Activating->Active over"
+           << " PG_STATE_ACTIVE=" << ps->state_test(PG_STATE_ACTIVE)
+           << " PG_STATE_WAIT=" << ps->state_test(PG_STATE_WAIT)
+           << " PG_STATE_PEERED=" << ps->state_test(PG_STATE_PEERED)
+           << " last_epoch_started=" << ps->info.last_epoch_started
```

Comment:
This is also in the PG psdout prefix already (les=)

Reply:
```
ostream &operator<<(ostream &out, const PeeringState &ps) {
    out << "pg[" << ps.info << " " << pg_vector_string(ps.up);
    if (ps.acting != ps.up) out << "/" << pg_vector_string(ps.acting);
    if (ps.is_ec_pg()) out << "p" << ps.get_primary();
    if (!ps.async_recovery_targets.empty()) out << " async=[" << ps.async_recovery_targets << "]";
    if (!ps.backfill_targets.empty()) out << " backfill=[" << ps.backfill_targets << "]";
    out << " r=" << ps.get_role();
    out << " lpr=" << ps.get_last_peering_reset();

    if (ps.deleting) out << " DELETING";

    if (!ps.past_intervals.empty()) {
        out << " pi=[" << ps.past_intervals.get_bounds() << ")/" << ps.past_intervals.size();
    }

    if (ps.is_peered()) {
        if (ps.last_update_ondisk != ps.info.last_update) out << " luod=" << ps.last_update_ondisk;
        if (ps.last_update_applied != ps.info.last_update) out << " lua=" << ps.last_update_applied;
    }

    if (ps.pg_log.get_tail() != ps.info.log_tail || ps.pg_log.get_head() != ps.info.last_update)
        out << " (info mismatch, " << ps.pg_log.get_log() << ")";

    if (!ps.pg_log.get_log().empty()) {
        if ((ps.pg_log.get_log().log.begin()->version <= ps.pg_log.get_tail())) {
            out << " (log bound mismatch, actual=[" << ps.pg_log.get_log().log.begin()->version << ","
                << ps.pg_log.get_log().log.rbegin()->version << "]";
            out << ")";
        }
    }

    out << " crt=" << ps.pg_log.get_can_rollback_to();

    if (ps.last_complete_ondisk != ps.info.last_complete) out << " lcod " << ps.last_complete_ondisk;

    out << " mlcod " << ps.min_last_complete_ondisk;

    out << " " << pg_state_string(ps.get_state());
    if (ps.should_send_notify()) out << " NOTIFY";

    if (ps.prior_readable_until_ub != ceph::signedspan::zero()) {
        out << " pruub " << ps.prior_readable_until_ub << "@" << ps.get_prior_readable_down_osds();
    }
    return out;
}
```